website/src/main/tut/docs/users/installation.md

```diff
@@ -12,6 +12,13 @@ Currently, Scalafix does not provide any IDE integrations with IntelliJ/ENSIME.
 
 ## sbt-scalafix
 
+_Since 0.6.0_
+
+_Later this plugin has `scalafixConfigure` method to configure scalafix for custom configuration like IntegrationTest. 
+Since 0.6.0 version it now sticks to more canonical way for such configurations. 
+The plugin has `scalafixConfigSettings` that can be imported to every configuration by `inConfig(_)`. 
```

Comment:
Can we remove all updates to the sbt-scalafix installation docs? I will can do it in a followup PR.

A general note, sentences like

> The plugin has `scalafixConfigSettings` that can be imported to every configuration by `inConfig(_)`. 

are very helpful if you are not aware of what `inConfig(_)` does. I generally prefer concrete examples like below since that is useful for both expert and beginner users.